<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CipherWidget</class>
 <widget class="QWidget" name="CipherWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cipher</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="message_label">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
        </font>
       </property>
       <property name="text">
        <string>Message</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPlainTextEdit" name="code_text_edit">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QSpinBox" name="key_spin_box">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="minimum">
          <number>-2147483648</number>
         </property>
         <property name="maximum">
          <number>2147483647</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="key_line_edit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
          </font>
         </property>
         <property name="text">
          <string>hill</string>
         </property>
         <property name="placeholderText">
          <string>key</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="affine_widget" native="true">
         <layout class="QGridLayout" name="affine_key_grid">
          <item row="1" column="0">
           <widget class="QLabel" name="key_b_label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>b =</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="key_a_label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>a =</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="key_a_spin_box">
            <property name="font">
             <font>
              <family>Ubuntu Mono</family>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="minimum">
             <number>-2147483648</number>
            </property>
            <property name="maximum">
             <number>2147483647</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="key_b_spin_box">
            <property name="font">
             <font>
              <family>Ubuntu Mono</family>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="minimum">
             <number>-2147483648</number>
            </property>
            <property name="maximum">
             <number>2147483647</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="code_label">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Code</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPlainTextEdit" name="message_text_edit">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="plainText">
        <string>the quick brown fox jumps over the lazy dog</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="key_label">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Key</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="cipher_label">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Cipher</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="alphabet_label">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Alphabet</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="cipher_combo_box">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="alphabet_line_edit">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
